include/GafferOSL/OSLCode.h

```diff
@@ -88,7 +88,7 @@ class GAFFEROSL_API OSLCode : public OSLShader
 		static size_t g_firstPlugIndex;
 
 		ShaderCompiledSignal m_shaderCompiledSignal;
-		std::unordered_map<Gaffer::ConstGraphComponentPtr, Gaffer::Signals::ScopedConnection> m_nameChangedConnections;
+		std::unordered_map<const Gaffer::GraphComponent*, Gaffer::Signals::ScopedConnection> m_nameChangedConnections;
```

Comment:
What compilation error do you get at IE? Ideally IE would be building with the same libs/options that we do, so there aren't any surprises. 

That said, I _think_ I was probably just being lazy in using a Ptr here. I don't think it's possible for the plug to die while it's still in this map, because to die it needs to be removed from its parent, and when that happens we remove it from the map in `parameterRemoved()`. Does that make sense?